src/platforms/common/data-management/event-grouping/grouping-enhancements.mdx

```diff
@@ -58,82 +119,80 @@ There are two types of **actions**: flag setting and setting variables.
 
-```
-function:myapplication::* +app
+```discover {tabTitle:Enhancement Rules}
+stack.function:myapplication::* +app
 ```
 
 You can also achieve the same result by marking other frames "not in-app." However, if that's the case, you should ensure that first all frames are set to "in-app" to override the defaults:
 
-```
-app:no             +app
-function:std::*    -app
-function:boost::*  -app
+```discover {tabTitle:Enhancement Rules}
+app:no                   +app
+stack.function:std::*    -app
+stack.function:boost::*  -app
 ```
 
 Forcing all frames to be in-app first is necessary because there might already have
 been some defaults set by the client SDK or earlier processing.
 
-**Cut Stack Traces**
+### Cut Stack Traces
 
 In many cases, you want to chop off the top or bottom of the stack trace. For instance, many code bases use a common function to generate an error. In this case, the error machinery will appear as part of the stack trace. For example, if you use Rust, you likely want to remove some frames that are related to panic handling:
 
-```
-function:std::panicking::begin_panic       ^-app -app ^-group
-function:core::panicking::begin_panic      ^-app -app ^-group
+```discover {tabTitle:Enhancement Rules}
+stack.function:std::panicking::begin_panic       ^-app -app ^-group
+stack.function:core::panicking::begin_panic      ^-app -app ^-group
 ```
 
 Here we tell the system that all frames from begin-panic to the crash location are not part of the application (including the panic frame itself). All frames above are also in all cases irrelevant for grouping.
 
 Likewise, you can also chop off the base of a stack trace. This is particularly useful if you have different main loops that drive an application:
 
-```
-function:myapp::LinuxMainLoop         v-group -group
-function:myapp::MacMainLoop           v-group -group
-function:myapp::WinMainLoop           v-group -group
+```discover {tabTitle:Enhancement Rules}
+stack.function:myapp::LinuxMainLoop         v-group -group
+stack.function:myapp::MacMainLoop           v-group -group
+stack.function:myapp::WinMainLoop           v-group -group
 ```
 
-**Stack Trace Frame Limits**
+### Stack Trace Frame Limits
 
 This isn't useful for _all_ projects, but it can work well for large applications with many crashes. The default strategy is to consider most of the stack trace relevant for grouping. This means that every different stack trace that leads to a crashing function will cause a different group to be created. If you do not want that, you can alternatively force the groups to be much larger by limiting how many frames should be considered.
 
 For instance, if any of the frames in the stack trace refer to a common external library, you could tell the system to only consider the top N frames, :
 
-```
+```discover {tabTitle:Enhancement Rules}
 # always only consider the top 1 frame for all native events
```

Comment:
This comment is a bit confusing. I think it means "consistently (or permanently) consider only the top 1 frame for all native events"